appvalidator/specs/webapps.py

```diff
@@ -95,6 +95,42 @@
     }
 }
 
+LANGUAGES_PROVIDED_OBJ = {
+    "expected_type": dict,
+    "process": lambda s: s.process_languages_provided,
+    "allowed_nodes": ["*"],
+    "not_empty": True,
+    "child_nodes": {
+        "*": {
+            "expected_type": dict,
+            "required_nodes": ["name", "version", "apps"],
```

Comment:
Uh, I added the tests, and all 3 seem to be required... Commenting any of them breaks one or more tests.